### PR TITLE
Grammar fix Readme

### PR DESCRIPTION
**Pull request by jermanuts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Notesnook is built using the following technologies:
 4. NPM — listed here because we **don't** use Yarn or PNPM or XYZ across any of our projects.
 5. Lerna & Nx — maintaining monorepos is hard but Nx makes it easier.
 
-> **Note: Each project in the monorepo contains its own architecture details which you can refer to.**
+> **Note: Each project in the monorepo contains its own architecture details, which you can refer to.**
```
Comment (thecodrr) [inline README.md:31]: Why the comma here?

Comment (jermanuts) [reply README.md:31]: adding a pause.

Comment (thecodrr) [reply README.md:31]: Why are we adding a pause here?

Comment (jermanuts) [reply README.md:31]: If you don't think it's needed, remove it. :+1:

Comment (thecodrr) [reply README.md:31]: I think we remove `, which you can refer to` completely.

```suggestion
> **Note: Each project in the monorepo contains its own architecture details.**
```

**Comment by thecodrr** [comment]
@jermanuts All looks fine but you forgot to Sign off your commits. Please go through the CONTRIBUTING guidelines.

**Pull request closed without merging**